enterprise/web.monitor/serversrc/org/netbeans/modules/web/monitor/server/NotifyUtil.java

```diff
@@ -117,28 +119,32 @@ void setIDE(String server) throws MalformedURLException {
     }
 		
     void sendRecord(MonitorData monData, String queryStr) {
-	
-	if(debug) log("NotifyUtil::notifyServer");  //NOI18N
-
-	if(ideServer != null) { 
-	    String urlStr = ideServer.concat(queryStr);
-	    if(debug) log("NotifyUtil: url is " + urlStr);  //NOI18N
-	    sendRecord(urlStr, monData); 
-	} 
-
-	if(otherIDEs.isEmpty()) return; 
-
-	Enumeration ides = otherIDEs.elements();
-	while(ides.hasMoreElements()) {
-
-	    String base = (String)ides.nextElement();
-
-	    if(debug) 
-		log("NotifyUtil: url is " + base.concat(queryStr)); //NOI18N
-	    if(!sendRecord(base.concat(queryStr), monData))
-		otherIDEs.remove(base); 
-	}
-	return; 
+        try {
+            if(debug) log("NotifyUtil::notifyServer");  //NOI18N
+
+            if(ideServer != null) {
+                    String urlStr = ideServer.concat(URLEncoder.encode(queryStr, "UTF-8"));
```

Comment:
nitpick: the UTF String literals need all the `//NOI18N`, right?

or: `StandardCharsets.UTF_8.name()`